src/scss/style.scss

```diff
@@ -1,14 +1,188 @@
+// Colors
```

Comment:
These variables should live in a seperate `settings` file, which can then be `@import`ed into this one.